### PR TITLE
263 email subject

### LINKED ISSUE #263: Change email subjects from "Sie" to "Du"

### PR DESCRIPTION
**Pull request by tillprochaska**
@roschaefer: Please do not merge yet, didn’t test everything.

* changes the plural for users to "Community", and the term for a singular user to "Mitglied".
* changes the nav bar item for requests to "Recherchen" while keeping "Frage"/"Fragen" in all other places.

Close #263

### CODE REVIEW

**Comment by drjakob** [comment]
Please give a status update

File: config/locales/de.yml [hunk 1]
```diff
@@ -2,27 +2,35 @@ de:
   application_name: 100eyes
   components:
     chat:
-      outdated_request: Du hast diese*r Lokalexpert*in bereits eine neue Frage gestellt. Nachfragen zu dieser Frage sind deshalb nicht mehr möglich.
+      outdated_request: Du hast diesem Mitglied bereits eine neue Frage gestellt. Nachfragen zu dieser Frage sind deshalb nicht mehr möglich.
```
Comment (roschaefer) [inline config/locales/de.yml:5]: Yet another term now? :unamused:

Comment (tillprochaska) [reply config/locales/de.yml:5]: Yes, decision by @FrauCsu and @drjakob

**Comment by roschaefer** [comment]
So, what's the matter? Can we merge this? @tillprochaska you can test emails here: http://localhost:3000/rails/conductor/action_mailbox/inbound_emails and see emails here: http://localhost:1080/#/

**Pull request merged**